meta-refkit-extra/doc/bluetoothspeaker.rst

```diff
@@ -0,0 +1,75 @@
+Bluetooth Speaker
+#################
+
+This is a demo to convert Intel 570x development board into stand
+alone bluetooth speaker. Main application is a python file
+``btspeaker.py`` and it is run under a Linux user called
+``btspeaker``. The demo uses fairly "stock" components like
+``PulseAudio``, ``BlueZ`` and ``ConnMan``. These are included in the
+IoT Reference OS Kit's gateway profile. The 570x doesn't have audio
+codec or speaker included so if you want to hear actual sound you need
```

Comment:
yes there really isn't a codec so putting jack or speaker would not help too much :)
But yes, I can try to rephrase. And yes monitor with hdmi and speakers should do fine.